gensim/models/doc2vec.py

```diff
@@ -778,6 +778,15 @@ def __str__(self):
             segments.append('t%d' % self.workers)
         return '%s(%s)' % (self.__class__.__name__, ','.join(segments))
 
+    def finished_training(self):
+        """
+        Discard parametrs that are used in training and score. Use if you're sure you're done training a model.
+        """
+        self._minimize_model(self.hs, self.negative > 0, True)
+        if hasattr(self, 'doctag_syn0'):
+            del self.doctag_syn0
```

Comment:
Many will consider the bulk-trained doctag-vectors a part of the model they want to retain.